plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/CodeWhispererFeatureConfigService.kt

```diff
@@ -116,6 +124,9 @@ class CodeWhispererFeatureConfigService {
         featureConfigs[name]?.value ?: FEATURE_DEFINITIONS[name]?.value
             ?: FeatureValue.builder().boolValue(false).build()
 
+    private fun connection(project: Project) =
+        ToolkitConnectionManager.getInstance(project).activeConnectionForFeature(QConnection.getInstance())
+
```

Comment:
i think codewhispererconnection is safer, not sure if population has aged out of that one yet

Reply:
If we are deprecating 233 I think it's probably safe by now